src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -0,0 +1,74 @@
+# Carbon Aware REST API
+
+## Endpoints
```

Comment:
I would recommend adding a table of contents for easier readability. There are VS Code plugins that do that for you.